src/ripple/overlay/impl/Tuning.h

```diff
@@ -52,13 +52,13 @@ enum
 
     /** How many milliseconds to consider high latency
         on a peer connection */
-    peerHighLatency     =  250,
+    peerHighLatency     =  300,
 
     /** How often we check connections (seconds) */
-    checkSeconds        =   10,
+    checkSeconds        =   32,
 
     /** How often we latency/sendq probe connections */
-    timerSeconds        =    4,
+    timerSeconds        =    8,
```

Comment:
`checkSeconds` would be fine if we also change `OverlayImpl::timer_count_` which also looks easy.  `seconds % seconds == seconds`, so also change `0` to `0s` in one place.